### ISSUE TITLE
请问youtube_match_model.py为何不用原论文的softmax的方式呢？

### ISSUE THREAD

**Issue by shuDaoNan9** [yangxudong/deeplearning#23]
我看YouTubeNet原本的召回阶段使用的是softmax分类，是因为认为这个应该被理解为多分类问题，可以“同时”选择多个“下个”要播放的视频吗？
https://github.com/yangxudong/deeplearning/blob/master/youtube_match_model/youtube_match_model.py
loss = tf.nn.sigmoid_cross_entropy_with_logits(
            labels=labels_one_hot,
            logits=logits)

谢谢！

**Comment by Zhangchangh** [comment]
#23

**Comment by Zhangchangh** [comment]
### 姓名
张长昊

### 本周工作
1.测试oneflow-documentation仓库中的脚本

### 下周工作
1.编译oneflow
2.协作完成#5545

**Comment by Zhangchangh** [comment]
### 姓名
张长昊

### 本周工作
1.测试oneflow-documentation仓库中的脚本

### 下周工作
1.编译oneflow
2.协作完成#1

**Comment by Zhangchangh** [comment]
### 姓名
张长昊

### 本周工作
1.测试oneflow-documentation仓库中的脚本

### 下周工作
1.编译oneflow
2.协作完成#1
